README.md

```diff
@@ -154,6 +155,75 @@ export default function () {
 }
 ```
 
+## Helpers
+
+The `xk6-kubernetes` extension offers helpers to facilitate common tasks when setting up a tests. All helper functions work in a implicit namespace to facilitate the development of tests segregated by namespace. The helpers are accessed using the following methods:
+
+|  Method      | Parameters|   Description |
+| -------------| ---| ------ |
+| helpers      | | returns helpers that operate in the default namespace |
+| namespacedHelpers | namespace | returns helpers that operate in the given namespace |
+
+
+The methods above return an object that implements the following helper functions:
+
+|  Method     | Parameters|   Description |
+| ------------ | --------| ------ |
+| createRandomNamespace| prefix         | creates a new namespace with the given prefix and returns its name.  |
+| getExternalIP        | service        | returns the external IP of a service if any is assigned before timeout expires|
+|                      | timeout in seconds | |
+| waitPodRunning | pod name | waits until the pod is in 'Running' state or the timeout expires. Returns a boolean indicating of the pod was ready or not. Throws an error if the pod is Failed. |
+|                | timeout in seconds | |
+| waitServiceReady         | service name | waits until the given service has at least one endpoint ready or the timeout expires |
+|                | timeout in seconds | |
+
+
+
+### Examples
+
+### Creating a pod in a random namespace and wait until it is running
+
+```javascript
+import { Kubernetes } from 'k6/x/kubernetes';
+
+let podSpec = {
+    apiVersion: "v1",
+    kind:       "Pod",
+    metadata: {
+        name:      "busybox",
+        namespace:  ""
+    },
+    spec: {
+        containers: [
+            {
+                name:    "busybox",
+                image:   "busybox",
+                command: ["sh", "-c", "sleep 30"]
+            }
+        ]
+    }
+}
+
+export default function () {
+  const kubernetes = new Kubernetes();
+
+  // create namespace with random name with prefix 'test-'
+  const ns = kubernetes.helpers().randomNamespace("test-")
+
+  // create pod in test namespace
+  pod.metadata.namespace = ns
+  kubernetes.create(pod)
+
+  // get helpers for test namespace
+  const helpers = kubernetes.namespacesHelpers(ns)
```

Comment:
Shouldn't this be `namespacedHelpers(ns)`? (note "namespaced" versus "namespaces".

Reply:
Yeap! thanks for catching this.